lib/Db/CollectiveShare.php

```diff
@@ -45,6 +54,7 @@ public function jsonSerialize(): array {
 			'token' => $this->token,
 			'owner' => $this->owner,
 			'editable' => $this->editable,
+			'password' => $this->password,
```

Comment:
Were does this get used / would it typically be used?

I'm wondering if we might want to leave the password hash out so it cannot leak - but then again we include the token, which we would not want to leak either. In logs I think the two will be filtered out.

Reply:
It's used in the frontend (sharing tab in sidebar, where shares are listed/configured), to determine if a password is set for the share. The files app does it exactly like this, but I agree that we could also just set a `hasPassword` flag in the `jsonSerialize()` function.

I don't think this will end up in logs, as its the payload that's sent as response body.